core/models/question.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from enum import Enum
+from . import Program, Form
+
+class Type(Enum):
```

Comment:
I want to be explicit when the question should render as 'select'

- there should be another field "input_type" that takes an enum choice, which is the type of element the dom should render.
- if it is 'select' (our most common field) , then we will include the options for that question in the response. 

other input_types include: radio, text, textarea, and date. we dont need an exhaustive list, these most common ones will do for now

Reply:
Yeah. I tried to see if I could pull the table types out of somewhere to be used instead of the enum, but I didn't find anything easily accessible. 

But for the `input_type` part of your comment, do you mean the current `Enum` should just have a `select` option that would direct the UI to query the `options` table?

Reply:
When i wrote that I was thinking two enums, but I think that is overkill. I know that the sheet means data type, But I think the more valuable thing to store on this table might be the input type the front end needs. 
- select
- text
- date
- textarea
- number
- radio (for booleans)

That way, its explicit for the frontend and it doesnt have to make inferences, just render.
I dont think the back end needs such specific data type definitions. It going to be text, a number, or a boolean. 

Does this make sense?

Reply:
I believe so. So we're not trying to have the column hold a specific SQL data type, but rather some generic data types for the input type. And we just want an `Enum` of the list in your previous comment?

Reply:
yes,  where the data is saved can handle the data type,  and we can also check with the future validators table.

Reply:
Changes pushed